resolvers/collectionPoint.js

```diff
@@ -1,87 +1,89 @@
-'use strict';
+"use strict";
 
-const db = require('../models');
+const db = require("../models");
 
 const collectionPointResolvers = {
-    Query: {
-        collectionPoints: () =>  db.collectionPoint.findAll(),
-        collectionPoint: (obj, args, context, info) => db.collectionPoint.findByPk(args.id)
-    },
-
-    collectionPoint: {
-        eventId: (obj, args, context, info) => db.event.findByPk(obj.eventId),
-        createdBy: (obj, args, context, info) => db.user.findByPk(obj.createdBy)
-    },
+  Query: {
+    collectionPoints: () => db.collectionPoint.findAll(),
+    collectionPoint: (obj, args, context, info) =>
+      db.collectionPoint.findByPk(args.id),
+    collectionPointsByEventId: (obj, args, context, info) =>
```

Comment:
nitpick: `collectionPointsByEvent` is a better format, no need to refer to the id in the query name
```suggestion
    collectionPointsByEvent: (obj, args, context, info) =>
```